applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py

```diff
@@ -37,6 +37,9 @@
 # from holohub.qcap_source import QCAPSourceOp
 from holohub.tool_tracking_postprocessor import ToolTrackingPostprocessorOp
 
+# Enable this line for vtk rendering
+# from holohub.vtk_renderer import VtkRendererOp
```

Comment:
Could you please also mention lines to uncomment in the README?

Reply:
Done! Updated README. :pray: